phpBB/includes/functions.php

```diff
@@ -5448,22 +5436,6 @@ function phpbb_user_session_handler()
 }
 
 /**
-* Check if PCRE has UTF-8 support
-* PHP may not be linked with the bundled PCRE lib and instead with an older version
-*
-* @return bool	Returns true if PCRE (the regular expressions library) supports UTF-8 encoding
-*/
-function phpbb_pcre_utf8_support()
```

Comment:
The function should be deprecated and moved to `functions_compatibility.php` first.
Then removed in phpBB 3.3.